src/main/java/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> FRUITS = new HashMap<>();
```

Comment:
```suggestion
    public static final Map<String, Integer> fruits = new HashMap<>();
```
this is not a magic number or something like this
the field can be just static final and not a constant (if it is not a "magic" data)